README.md

```diff
@@ -27,6 +27,14 @@ var myMap map[ComplexKeyType]string
 f.Fuzz(&myMap) // myMap will have exactly one element.
 ```
 
+Customize a string within one encoding range([Unicode](https://en.wikipedia.org/wiki/Plane_(Unicode)#Basic_Multilingual_Plane)). 
```

Comment:
This chunk looks good now, but I still think it needs to go after we introduce the custom fuzz function concept, can you move it down to line [67](https://github.com/google/gofuzz/pull/42/files#diff-04c6e90faac2675aa89e2176d2eec7d8L67)?

Reply:
I think of what you said before, it is too early to introduce this custom fuzz function concept. 

At the same time, I also found that on line 67 of the README, it is not easy to add an introduction of generating a string in the custom encoding range, so delete it here.

Give back the choice to the developer, who can find related usage in the `example_test.go`.